src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -76,6 +77,32 @@ class RawTableRelation(
     cursors.map(rawClient.filterOnePartition(filter, _, limit))
   }
 
+  def pullFromKeysSeq(
+      keys: List[String],
+      get: String => IO[RawRow]
+  ): Pull[IO, RawRow, Unit] =
+    keys match {
+      case Nil => Pull.done
+      case key :: tail =>
+        Pull.eval(get(key)).flatMap { row =>
+          Pull.output(Chunk.singleton(row)) >>
+            pullFromKeysSeq(tail, get)
+        }
+    }
+
+  def getStreamsByKeys(keys: Array[String])(
+      client: GenericClient[IO],
+      limit: Option[Int],
```

Comment:
If we assume the number of keys will be relatively small, and limited to be small by us, we can safely ignore both `limit: Option[Int]` and `numPartitions`, I think. Maybe use `limit: Option[Int]` on the set of keys, somewhere, but it's more of a hint anyway so perhaps we don't even need to bother with that.

Reply:
limit was already ignored... now `numPartitions` is used for the maximum parallelism of parEvalMap, we could put a reasonable cap (or fixed parallelism instead) of 4 - 10 if we are worried about users setting a high partition count